package.json

```diff
@@ -195,6 +229,7 @@
     "npm": "^6.14.5",
     "typescript": "^2.6.2",
     "vsce": "^1.75.0",
-    "vscode": "^1.0.0"
+    "vscode": "^1.0.0",
+    "vscode-tmgrammar-test": "0.0.10"
```

Comment:
This is only a testing dependency, so I don't think it should be included here.

Reply:
Sure thing: while I was testing, it seemed like it would keep trying to reinstall this (because of the change to `test.sh` that invokes it with `npx`, I guess). Adding it to the package manifest solved that, but it's not necessary for development. Even without the additional dev dependency, it still functions correctly using `npx`, so I'm happy to drop it